apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerConnectionStringBuilder.cs

```diff
@@ -88,6 +89,36 @@ public string CredentialFile
             set => this[CredentialFileKeyword] = value;
         }
 
+        /// <summary>
+        /// Option to change between the default handling of null database values (return <see cref="DBNull.Value"/>) or
+        /// the legacy handling (return the default value for whatever type is requested).
+        /// </summary>
+        /// <remarks>
+        /// <para>
+        /// If this is <c>false</c> (the default), requesting a value from a <see cref="SpannerDataReader"/> that is null
+        /// in the database will return <see cref="DBNull.Value"/>, which may cause an <see cref="InvalidCastException"/> if
+        /// the requested type is not compatible with that value.
+        /// </para>
+        /// <para>
+        /// If this is <c>true</c>, requesting a value from a <see cref="SpannerDataReader"/> that is null in the
+        /// database will return the default value of the requested type (e.g. 0 or a null reference). This is the legacy
+        /// behavior from release 1.0 of this package.
+        /// </para>
+        /// <para>
+        /// This property only sets the behavior for top-level values. For arrays and structs, null references and default values
+        /// are still used if the array or struct field value is null; this provides a better general experience, allowing
+        /// conversions to specific array types for example.
+        /// </para>
+        /// <para>
+        /// This property corresponds with the value of the "UseClrDefaultForNull" part of the connection string.
+        /// </para>
+        /// </remarks>
+        public bool UseClrDefaultForNull
+        {
+            get => GetValueOrDefault(UseClrDefaultForNullKeyword).Equals("True", StringComparison.OrdinalIgnoreCase);
+            set => this[UseClrDefaultForNullKeyword] = value ? "True" : "False";
```

Comment:
`value.ToString()` should work here. It is [hard coded](https://github.com/dotnet/corefx/blob/a00ae9072b9244d2fa7ee282ec90a5d8500704ac/src/Common/src/CoreLib/System/Boolean.cs#L85) to return these literals.

Reply:
Right. I wasn't sure whether it was localized. Will change to use Value.ToString(), but with an explanatory comment.